artifactory/commands/transferfiles/status.go

```diff
@@ -106,3 +109,7 @@ func sizeToString(sizeInBytes int64) string {
 	}
 	return fmt.Sprintf("%.1f %ciB", float64(sizeInBytes)/float64(divider), sizeUnits[sizeUnitIndex])
 }
+
+func getRepoStateMissingErrorMsg(repoKey string) string {
```

Comment:
Since this function is used only once, can we remove it and instead use the below?
```
errorutils.CheckErrorf("Could not find the state file of repository '%s'. Aborting.", repoKey)
```